app/server.go

```diff
@@ -210,6 +213,11 @@ func NewServer(options ...Option) (*Server, error) {
 	// Use this app logger as the global logger (eventually remove all instances of global logging)
 	mlog.InitGlobalLogger(s.Log)
 
+	// It is important to initialize the hub only after the global logger is set
+	// to avoid race conditions while logging from inside the hub.
+	fakeApp := New(ServerConnector(s))
```

Comment:
I know @jespino is working on eliminating fakeApp. It would be nice if you where to move the hub code to avoid its use here.

Reply:
Yes, Jesus and I are aware of this. It's not possible to move the hub right away from the app layer because it's a bit tightly coupled. But it's on our plan to work on it in small chunks.